framework/src/vectorpostprocessors/VectorPostprocessorData.C

```diff
@@ -15,10 +15,24 @@ VectorPostprocessorData::VectorPostprocessorData(FEProblemBase & fe_problem)
 {
 }
 
+void
+VectorPostprocessorData::init(const std::string & /*name*/)
+{
+}
```

Comment:
I added this because I was using it in the first implementation. It makes it consistent with `PostprocessorData`. It's now empty (as of the last commit). I'm fine leaving it in, but it can also be removed again.

Reply:
I'm ok with it there